.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+The template below is mostly useful for bug reports and support questions.
+Feel free to remove anything which doesn't apply to you and add more information where it makes sense.
+
+# Required information
+
+ * Distribution:
+ * Distribution version:
+ * Kernel version:
+ * LXC version:
+ * LXD version:
+ * Storage backend in use:
```

Comment:
What if we put `output of lxc info $remote` here? That'll give us all of this modulo the distro info, I think.